### PR TITLE
created outline for LCD documentation

### PR DESCRIPTION
**Pull request by jyanev**

### CODE REVIEW

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc [hunk 1]
```diff
@@ -1,7 +1,94 @@
...
+ifndef::rootpath[]
+:rootpath: ../../
+endif::rootpath[]
+
+ifdef::rootpath[]
+:imagesdir: {rootpath}{imagesdir}
+endif::rootpath[]
+
+
 ==== LCD
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc:12]: This is LCD 1602 module which is different from standard LCD. Rename it to LCD 1602

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc [hunk 2]
```diff
@@ -1,7 +1,94 @@
...
 [.text-right] 
 https://github.com/oss-slu/Pi4Micronaut/edit/develop/micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc[Improve this doc]
 
+===== Overview
+This section provides details of the LCD (Liquid Crystal Display) circuit, including its components, assembly instructions, and functionality. The LCD is a M × N pixel screen that can be used to display different images or words.
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc:17]: Overview should be of LCD 1602. LCD1602, or 1602 character-type liquid crystal display, is a kind of dot matrix module to show letters, numbers, and characters and so on. It's composed of 5x7 or 5x11 dot matrix positions; each position can display one character. There's a dot pitch between two characters and a space between lines, thus separating characters and lines. The number 1602 means on the display, 2 rows can be showed and 16 characters in each.

File: micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc [hunk 3]
```diff
@@ -1,7 +1,94 @@
...
+
+===== Components
+. LCD
```
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/components/outputComponents/lcdScreen.adoc:20]: Change it to LCD1602

File: micronautpi4j-utils/src/docs/asciidoc/img/lcd-circuit.png
Comment (ruthvikm) [inline micronautpi4j-utils/src/docs/asciidoc/img/lcd-circuit.png:1]: Rename it

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Change the LCD to LCD1602 where ever it is required

**Review by SwindleA — CHANGES_REQUESTED** [review]

**Review by SwindleA — APPROVED** [review]
Looks good

**Review by ruthvikm — APPROVED** [review]
Looks good

**Pull request merged**